### PR TITLE
Angular Migration from v14 to v15 

### PR DESCRIPTION
**Pull request by sedin-tushar**
- Angular Version Update: The Angular framework and dependencies has been updated from version 14 to  15.

### CODE REVIEW

File: projects/content-section/package.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
   "name": "@project-sunbird/sb-content-section",
   "version": "8.0.0",
   "peerDependencies": {
```
Comment (princegupta1131) [inline projects/content-section/package.json:4]: @sedin-tushar change the version

File: README.md [hunk 2]
```diff
@@ -53,7 +53,9 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
 | release-5.1.0_v12 |      5.1.2          |     Ng V12      |
 | release-6.0.0_v13 |      6.0.0          |     Ng V13      |
 | release-7.0.0     |      7.0.0          |     Ng V13      |
-| 8.0.0_14          |      8.0.0          |     Ng V14    |
+| 8.0.0_14          |      8.0.0          |     Ng V14      |
+| 8.0.0_v15          |      8.0.1          |     Ng V15      |
```
Comment (princegupta1131) [inline README.md]: @sedin-tushar  fix the table alignment

File: package.json [hunk 3]
```diff
@@ -12,44 +12,44 @@
...
+    "protractor": "7.0.0",
+    "ts-node": "8.3.0",
+    "tslint": "6.1.0",
+    "typescript": "4.9.4"
   },
   "engines": {
-    "node": ">=16.19.0",
-    "npm": ">=8.19.3"
-  }
+    "node": ">=18.19.0",
```
Comment (princegupta1131) [inline package.json]: why u have added 18

Comment (sedin-tushar) [reply package.json]: it works with " >=16.19.0 <=18.19.0",i will change this

**Review by rajeevsathish — APPROVED** [review]

**Pull request merged**